### ISSUE TITLE
Must wait 1 minute after creating new direct chat channel to actually send chat messages

### ISSUE THREAD

**Issue by cf-lam** [GetStream/stream-chat-swift#996]
## What did you do?
Create a direct chat channel. Call channelController synchronize(). 

## What did you expect to happen?

I can start a direct chat channel and start chatting instantly.

## What happened instead?

The synchronize() call takes 1 minute to complete, and logs an error message, after which I can send messages. Any messages sent before this logging statement or completion of synchronize() results in any sending of messages to disappear.

2021-04-15 12:02:20.980866-0500 CommunityApp[61138:17443853] Connection 3: missing error, so heuristics synthesized error(1:53)
2021-04-15 12:02:20.981053-0500 CommunityApp[61138:17443853] Connection 3: encountered error(1:53)
2021-04-15 12:02:20.987005-0500 CommunityApp[61138:17448707] Task <17095AA5-C681-4505-8BA1-9D23DBC47D9D>.<1> finished with error [57] Error Domain=NSPOSIXErrorDomain Code=57 "Socket is not connected" UserInfo={NSErrorFailingURLStringKey=wss://chat-proxy-dublin.stream-io-api.com/connect?api_key=&json=%7B%22server_determines_connection_id%22:true,%22user_id%22:%2221431%22,%22user_details%22:%7B%22id%22:%2221431%22%7D%7D, NSErrorFailingURLKey=wss://chat-proxy-dublin.stream-io-api.com/connect?api_key=&json=%7B%22server_determines_connection_id%22:true,%22user_id%22:%2221431%22,%22user_details%22:%7B%22id%22:%2221431%22%7D%7D, _NSURLErrorRelatedURLSessionTaskErrorKey=(
"LocalWebSocketTask <17095AA5-C681-4505-8BA1-9D23DBC47D9D>.<1>"
), _NSURLErrorFailingURLSessionTaskErrorKey=LocalWebSocketTask <17095AA5-C681-4505-8BA1-9D23DBC47D9D>.<1>}
2021-04-15 12:02:20.981 [ERROR] [io.getStream.chat.core.web_socket_engine_queue] [URLSessionWebSocketEngine.swift:62] [doRead()] > Failed receiving Web Socket Message with error: Error Domain=NSPOSIXErrorDomain Code=57 "Socket is not connected" UserInfo={NSErrorFailingURLStringKey=wss://chat-proxy-dublin.stream-io-api.com/connect?api_key=&json=%7B%22server_determines_connection_id%22:true,%22user_id%22:%2221431%22,%22user_details%22:%7B%22id%22:%2221431%22%7D%7D, NSErrorFailingURLKey=wss://chat-proxy-dublin.stream-io-api.com/connect?api_key=&json=%7B%22server_determines_connection_id%22:true,%22user_id%22:%2221431%22,%22user_details%22:%7B%22id%22:%2221431%22%7D%7D}

The DemoApp doesn't suffer from this problem, so I wanted to determine if it's backend issue related to my keys or misconfiguring the environment. When I swap out the API keys and start a direct chat with a user in the DemoApp environment using my code to start direct chat, I don't experience this problem.

## GetStream Environment
**GetStream Chat version:**
**GetStream Chat frameworks:** StreamChat, 'StreamChatUI', '~> 3.2.0-beta.6', StreamChatClient, StreamChatCore
**iOS version:**
**Swift version:**
**Xcode version:**
**Device:**

**Comment by cjmconie** [comment]
I am experiencing this issue too.

```swift
/// 1: Use the `ChatClient` to create a `ChatChannelController` with a list of user ids
let channelController = try ChatClient.shared.channelController(
    createDirectMessageChannelWith: [user.id],
    isCurrentUserMember: true,
    name: nil,
    imageURL: nil,
    extraData: .defaultValue
)

/// 2: Call `ChatChannelController.synchronize` to create the channel.
channelController.synchronize { error in
    if let error = error {
        /// 4: Handle possible errors
        print(error)
    }

    // ... present direct chat channel vc
}
```

## GetStream Environment

GetStream Chat version: 3.2.0-beta.6
GetStream Chat frameworks: StreamChatClient, StreamChatCore, StreamChat
iOS version: 13.0
Swift version: 5.2.3
Xcode version: 12.4
Device: Any

**Comment by VojtaStavik** [comment]
Hi, thank you for your reports. I can confirm we're experiencing some issues with the web socket connection. We're investigating this.

**Comment by avaitla** [comment]
This is a show stopping issue for us and an absolutely unfortunate member experience.

**Comment by VojtaStavik** [comment]
I believe this issue is resolved in `v3.1.8` and `v3.2.0-beta.8`. Can you upgrade to these versions and let me know if you still experience the issue?

**Comment by VojtaStavik** [comment]
@avaitla @cf-lam I'm closing this issue since it should be fixed in `v3.1.8` and `v3.2.0-beta.8`. Please re-open in case you're experiencing the problem event after upgrading.
